README.md

```diff
@@ -23,15 +25,21 @@ These files can be used to deploy your plugins.
 
 Deployment
 ----------
+
 You can deploy your plugins using the CDAP CLI:
 
+```
     > load artifact <target/sftp-actions-<version>.jar config-file <target/sftp-actions-<version>.json>
+```
 
 For example, if your artifact is named 'sftp-actions-1.0.0':
 
+```
     > load artifact target/sftp-actions-1.0.0.jar config-file target/sftp-actions-1.0.0.json
+```
 
-## Mailing Lists
+Mailing Lists
```

Comment:
I don't think we ever decided on anything, but I've been using RST in the meantime for consistency with the other PRs I've submitted.